index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+    <main>
+        <section class="container">
+                <nav class="nav_bar">
+                        <span class="myLogo"><a href="#"><img src="image/My_Logo.svg" alt="My Logo"></a></span>
+                        <span class="contact"><a href="#"><img src="image/Icon - Menu.svg" alt="Contact Us"></a></span>
```

Comment:
You have well specified an `alt` attribute for the hamburger image. Instead of "Contact Us" I suggest that you use an alt value that relates more to its usage since it's a hamburger menu. e.g "mobile menu"

Also, the hamburger menu isn't a link to any part of the page or an external page, rather it would display the mobile menu when it's clicked. I suggest that you remove the anchor tag that it's enclosed with.